assets/css/_fantasy_teams.scss

```diff
@@ -98,6 +98,18 @@
   }
 }
 
+.fantasy-team-table {
+
+  th.points,
+  td.points {
+    text-align: right;
+  }
```

Comment:
Rule declaration should be followed by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)

Reply:
fixed